server/filesystem.go

```diff
@@ -265,6 +281,8 @@ func (fs *Filesystem) getCachedDiskUsage() (int64, error) {
 	fs.lastLookupTime = time.Now()
 	atomic.StoreInt64(&fs.diskUsage, size)
 
+	fs.lookupInProgress = false
```

Comment:
For safety, this should probably be a deferred call up where it is set so if this function ever changes or manages to exit early down the road we don't forget to set this back to false.